### PR TITLE
Intentionally run draft PR CI by comment

### PR DESCRIPTION
**Pull request by Ikuni17**
In an effort to reduce CI minute usage on Kibana, we want to only trigger draft PRs by intentional comment: `buildkite test this`.

### CODE REVIEW

File: src/pullRequests.ts [hunk 1]
```diff
@@ -497,6 +497,10 @@ export default class PullRequests {
 
     // If this trigger was from a commit or initial PR creation, and the above checks didn't fail, we should trigger a build
     if (context.type === PullRequestEventTriggerType.Create || context.type === PullRequestEventTriggerType.Update) {
+      if (context.pullRequest.draft) {
```
Comment (Ikuni17) [inline src/pullRequests.ts:500]: Does this need to be restricted to Kibana repo only as well?

File: src/models/prConfig.ts [hunk 2]
```diff
@@ -7,6 +7,7 @@ export class PrConfig {
   allow_org_users = false;
   build_on_comment = true;
   build_on_commit = true;
+  build_drafts = true;
```
Comment (mistic) [inline src/models/prConfig.ts:10]: I think we should extend on `trigger_comment_regex` to also support the `/ci` command probably with 

```
trigger_comment_regex = '^(?:(?:buildkite\W+)?(?:build|test)\W+(?:this|it))|^retest$|^\/ci$';
```

Comment (mistic) [reply src/models/prConfig.ts:10]: We've agreed offline to do it otherwise for now

**Review by mistic — APPROVED** [review]
lgtm

**Pull request merged**